addons/arsenal/defines.hpp

```diff
@@ -19,6 +19,9 @@
 #define ASCENDING 0
 #define DESCENDING 1
 
+// Favorites
+#define FAVORITES_COLOR (GVAR(favoritesColor) + [1])
```

Comment:
Since alpha is modified for the right panel (in case the item doesn't fit), I opted for this solution instead, but it's a bit dirty.